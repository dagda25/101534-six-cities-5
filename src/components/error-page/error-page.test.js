import React from "react";
import renderer from "react-test-renderer";
import ErrorPage from "./error-page";


describe(`Render ErrorPage`, () => {
  it(`Render ErrorPage`, () => {
    const tree = renderer
      .create(
          <ErrorPage />,
          {
            createNodeMock: () => {
              return {};
            }
          }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
