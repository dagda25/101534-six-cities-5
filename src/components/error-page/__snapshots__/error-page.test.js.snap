// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ErrorPage Render ErrorPage 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="/img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            />
          </ul>
        </nav>
      </div>
    </div>
  </header>,
  <main
    className="page__main page__main--favorites page__main--favorites-empty"
  >
    <div
      className="page__favorites-container container"
    >
      <section
        className="favorites favorites--empty"
      >
        <h1
          className="visually-hidden"
        >
          Server Error
        </h1>
        <div
          className="favorites__status-wrapper"
        >
          <b
            className="favorites__status"
          >
            Server Error
          </b>
          <p
            className="favorites__status-description"
          >
            Please try again later
          </p>
        </div>
      </section>
    </div>
  </main>,
  <footer
    className="footer container"
  >
    <a
      className="footer__logo-link"
      href="main.html"
    >
      <img
        alt="6 cities logo"
        className="footer__logo"
        height="33"
        src="img/logo.svg"
        width="64"
      />
    </a>
  </footer>,
]
`;
